packages/devextreme/testing/helpers/pointerMock.js

```diff
@@ -151,6 +151,13 @@
                 return this;
             },
 
+            _wheel: (config) => {
+                triggerEvent('wheel', config);
+
+                // eslint-disable-next-line spellcheck/spell-checker
+                inferno.rerender();
```

Comment:
what is this for?

Reply:
I believe it's not necessary now because actual scrollview tests should not be run in renovation